### ISSUE TITLE
Control Icons aren't visible in IOS9

### ISSUE THREAD

**Issue by Maqsyo** [sampotts/plyr#1104]
### Expected behaviour
All icons are visible

### Actual behaviour
Only one play-button is visible
![img_0042](https://user-images.githubusercontent.com/19644698/42925975-469e4580-8b30-11e8-9de7-4bd59fb6f3ac.PNG)

### Environment
Plyr version: 3.3.22
Browser: Safari for iOS
Version: Default
Operating System: iOS
Version: 9

**Comment by sampotts** [comment]
Are you seeing this on https://plyr.io too? Any errors in the console (debugging via Safari)?

**Comment by Maqsyo** [comment]
Yeah the same behavior on plyr.io and no console logging.

The "use" parameters are empty in the inspector, but this seems to be fine, because the play-button renders too

**Comment by friday** [comment]
Does it happen in incognito mode?

**Comment by Maqsyo** [comment]
We cleared the cached and the bug still appears. I'm sorry, but the code have to be different for these icons.

I think it is hard for you to check this error, without an iOS9 Device. But i will test everything for you as fast as possible.

**Comment by sampotts** [comment]
> I have this problem in iOS 11.4.1 and latest version of safari in mac

On https://plyr.io ? Works ok for me. If that’s ok your own implementation, it’s likely an issue with your configuration.

**Comment by mostafaznv** [comment]
I checked it, but problem is same.
I downloaded `plyr.svg` and set local address in player configuration, problem solved

**Comment by drsmall** [comment]
I still have this issue. Except not even the play button appears for me. 

@mostafaznv did you do anything else to get it to work? I tried pointing the iconUrl to the local plyr.svg but no luck.

**Comment by mostafaznv** [comment]
> I still have this issue. Except not even the play button appears for me.
> 
> @mostafaznv did you do anything else to get it to work? I tried pointing the iconUrl to the local plyr.svg but no luck.

no. I just addressed svg locally.

**Comment by danjaywing** [comment]
@mostafaznv Can you confirm *exactly* what you did to get this working? I too am pointing the code to the local svg - it works everywhere BUT IOS.

This is very frustrating.

**Comment by danjaywing** [comment]
@sampotts The icons are also missing on plyr.io, this isn't a cache issue, or a local implementation issue.

**Comment by sampotts** [comment]
They show up fine for me on latest iOS. What version are you testing with?

**Comment by danjaywing** [comment]
Correct, iOS9, which accounts for 19.5% of all iPads according to the stats here: https://david-smith.org/iosversionstats/

**Comment by danjaywing** [comment]
I believe this is the same issue as reported here #1191, only I've tried the version that the fix is supposedly applied to and I'm having no luck.

I'm not convinced it's the loading of the sprite, rather it's the handling of the sprite.

**Comment by danjaywing** [comment]
Confirmed, issue present in Safari 9.1

**Comment by danjaywing** [comment]
@sampotts I checked in safari 9.1 and the SVGs are still using `href` as opposed to `xlink:href`, this in spite of the line of code you added to resolve it

**Comment by danjaywing** [comment]
Yet this article suggests it's the caching mentioned in other tickets:

https://www.smashingmagazine.com/2016/05/safari-svg-sprite-bug/

I'm at a loss, I need this resolving as iOS9 is very much still a thing, and it's a shame as plyr is generally a great solution.

**Comment by sampotts** [comment]
It's a shame we can't get some more official numbers. One fellas app is a bit of a small sample set. 

Anyway, I did some digging last night but could not get the SVGs to render in iOS 9. The SVGs are built dynamically and injected with both `href` and `xlink:href` but even with just the later, it still doesn't render them. I've tried a few different methods but can't seem to get a working method. I'll keep looking.

**Comment by danjaywing** [comment]
Well with the latter link I'm not convinced you'll get it working with svg sprites, the reason OP's big play button loads is because it's loading the sprite in once and then locking it away in cache preventing the others from loading.

Seems the only way around it is to inline the svgs.

You're using svg4everybody right? Might there be a way of tricking it into thinking SVG isn't supported in iOS9 so that it renders the fallback images?

Or could you detect and render fallbacks yourself?

**Comment by RemusVasii** [comment]
The SVG icons became visibile on my iOS9 after removing this from plyr.css:

.plyr--video .plyr__control svg {
	filter: drop-shadow(0 1px 1px rgba(0, 0, 0, .15))
}
